recipes/gsl-lite/all/conanfile.py

```diff
@@ -24,7 +24,7 @@ class GslLiteConan(ConanFile):
     #  3. GSL_UNENFORCED_ON_CONTRACT_VIOLATION: nothing happens
     #
     options = {
-        "on_contract_violation": ["terminate", "throw", "unenforced"]
+        "on_contract_violation": ["terminate", "throw", "unenforced", "None"]
```

Comment:
None is not a good name in this case, it gets too close to options being able to not be defined